index.html

```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="cards_container">
+            <div class="card_img">
+                <img src="./images/SnapshootPortfolio(2).svg" alt="card image">
+            </div>
+            <div class="card_text">
+                <h1>Tonic</h1>
+                <div class="card_text_flex">
+                    <p><strong>CANOPY</strong></p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>Band End Dev</p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>2015</p>
+                </div>
+                <p class="card_paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="card_lang_flex">
+                    <p><strong>html</strong></p>
+                    <p><strong>css</strong></p>
+                    <p><strong>javascript</strong></p>
+                </div>
+                <div class="card_button">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+
+
+
+        <div class="cards_container">
+            <div class="card_img">
+                <img src="./images/SnapshootPortfolio(3).svg" alt="card image">
+            </div>
+            <div class="card_text">
+                <h1>Multi-Post Stories</h1>
+                <div class="card_text_flex">
+                    <p><strong>CANOPY</strong></p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>Band End Dev</p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>2015</p>
+                </div>
+                <p class="card_paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="card_lang_flex">
+                    <p><strong>html</strong></p>
+                    <p><strong>css</strong></p>
+                    <p><strong>javascript</strong></p>
+                </div>
+                <div class="card_button">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+
+
+
+    </section>
+
+    <section class="about_section">
+        <div class="about_title">
+            <h1>About Myself</h1>
+        </div>
+        <div class="about_descript">
+            <p> I’m a software developer! I can help
+                you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="about_connect_links">
+            <a href="#">LET'S CONNECT</a>
+            <ul>
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/160082656-64006d72-79c3-40a5-8c56-8ca4bb8aaa77.png) | ![image](https://user-images.githubusercontent.com/10332499/160082541-769e921b-4fe7-478e-b65b-dd0ae7bd7761.png)